packages/hash/external-services/kratos/kratos.yml

```diff
@@ -0,0 +1,61 @@
+# Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
+
+version: v0.7.1-alpha.1
+
+dsn: memory
```

Comment:
Is this the data source for kratos? In that case should this be the Postgres database and not memory?

Reply:
Yes, good spot that should be removed. I think I didn't notice this because the environment variable set in the docker compose file was overriding this setting (in my dev environment the postgres database was used to persist the kratos identities/sessions/etc).

Have removed this in https://github.com/hashintel/hash/pull/889/commits/0d166a96f8836e8d34af318001dc3a463272ff3e